ui/src/app/views/breadcrumb/component.ts

```diff
@@ -86,13 +86,23 @@ export class BreadcrumbComponent implements OnInit, OnDestroy {
         label = child.snapshot.data['cohort'].name;
       }
       if (label === 'Param: Concept Sets Name') {
-        label = child.snapshot.data['workspace'].name + ' Concepts';
+        label = child.snapshot.data['workspace'].name;
```

Comment:
I'd drop this clause entirely, the original motivation to dig into the breadcrumbs is that the concept sets list breadcrumb logic didn't match the rest of the application, should drop the "param" piece for the list page in app-routing-module.ts